### ISSUE TITLE
Compact links

### ISSUE THREAD

**Issue by notuntoward** [studiowebux/obsidian-spotify-link#9]
I noticed that when I link to a bunch of songs or many passages in a podcast, that the quoted links and extra meta information chew up a lot of vertical space.  

**Update**: I just looked at the updated documentation: I should be able to shrink the links using templates, but not, as far as I can tell, by removing the quotes.  But are these working yet?  I tried this podcast table template:

`| {{ episode_name }} | {{ timestamp }}|`

but here's the link I got:

```
> ['**A legally binding treaty to fight the plastic problem**': ***The plastic problem is everywhere: in our oceans, communities, even inside our bodies. Plastic is abundant and very cheap, and the amount we produce is expected to triple by 2060 from 430 million tons a year to 1.2 billion tons, according to the OECD. The large amount of plastic could produce four billion tons of greenhouse gases, so a fix is becoming increasingly necessary. This week, Bloomberg Green senior reporter Akshat Rathi sat down with Inger Andersen, head of the UN’s Environment Program, to talk about an upcoming treaty that tackles increasing levels of plastic. You can read the transcript of this conversation here. Zero is a production of Bloomberg Green. Our producer is Oscar Boyd and our senior producer is Christine Driscoll. Special thanks to Kira Bindrim, Leslie Kaufman and Tiffany Tsoi. Email us at zeropod@bloomberg.net. For more coverage of climate change and solutions, visit https://www.bloomberg.com/green.See omnystudio.com/listener for privacy information.***, released 2024-03-28 | **15s** (1%)](https://open.spotify.com/episode/70xMHMhnQZz6YzWBN6trXY)
> Sat Mar 30 2024 - 1:08:55 PM
```

**Comment by studiowebux** [comment]
Hello ! 

I didn't test all of them for the episode. (if some of them does not resolve let me know, by not resolving it will print the {{ ... }} as is instead of replacing)

but in summary, _I personnaly only listen to song (so never tested the podcast before yesterday)_
so the `append <track or episode> with timestamp` was made especially for the songs and there is no configurable template for that one (the with timestamp I mean), 
i think that is why you don't see the changes.
I will check to integrate the template with that one as well. I think i decoupled the features correctly yesterday.

I will try to do that and test it within the next few days, I will update this ticket as soon as I get time to do it 😊

TL;DR:
The append with timestamp has a hardcoded template.

So if my understanding is good
you need to short the description length ? this one might be harder to do as I dont have anything currently to configure the text length

**Comment by studiowebux** [comment]
<img width="1210" alt="image" src="https://github.com/studiowebux/obsidian-spotify-link/assets/37277136/53a0ceba-ee72-4ad3-9618-c7403f496d6e">

the first one is hardcoded 

the second one uses the template

**Comment by notuntoward** [comment]
Thanks, yes, that's right.  For podcasts, at least for me, short links are better.  The reason is that, if I'm listening to one with details I want to remember, I'll take a note as it plays, making a bullet with a short description and a timestamp.

For music, it's the other way around: the whole song is what's interesting, and then a big link is luxurious.

**Comment by notuntoward** [comment]
Yes, those capabilities should work well.  Looks like I can edit them as needed.  Thanks!
